docs/source/command_line.rst

```diff
@@ -593,6 +593,41 @@ in error messages.
 
     Show absolute paths to files.
 
+.. option:: --ignore-error-codes CODES
+
+    This flag makes mypy ignore all errors with given error codes. Flag accepts
```

Comment:
I think that we should include a link to documentatino about error codes here instead of later. The text "error codes" could link to the relevant topic.

Grammar nit: "Flag" -> "This flag"